internal/agent/patch_sender.go

```diff
@@ -207,7 +216,19 @@ func (p *patchSenderIngest) sendAllDeltas(allDeltas []inventoryapi.RawDeltaBlock
 			return err
 		}
 
-		p.lastConnection = currentTime
+		if err = p.store.UpdateAndSaveLastSuccessSubmission(); err != nil {
+			llog.WithError(err).Error("can't save submission time")
```

Comment:
Could this log be in the save function?

Reply:
do you mean within `saveLastSuccessSubmission()`? why?

I usually rather logging when handling errors

Reply:
I agree, only I found it strange that the handling is only for logging. 
But I'm okey with that 👍